src/LibraBFT/Impl/Consensus/BlockStorage/BlockTree.agda

```diff
@@ -15,14 +15,16 @@ open import LibraBFT.ImplShared.Consensus.Types
 open import LibraBFT.ImplShared.Interface.Output
 open import LibraBFT.ImplShared.Util.Crypto
 open import LibraBFT.ImplShared.Util.Dijkstra.All
+open import Level
 open import Optics.All
 open import Util.ByteString
 open import Util.Hash
 import      Util.KVMap                                           as Map
 open import Util.PKCS
 open import Util.Prelude
+open import Dijkstra.AST.Branching
 open import Dijkstra.AST.Core
-open import Dijkstra.AST.Either renaming (EitherD to EitherAST)
+import      Dijkstra.AST.Either as EitherAST renaming (EitherD to EitherAST)
```

Comment:
Yes, there are two kinds of historical inconsistencies:

- the difference between `RWS` and `EitherD` (the latter due to `Either` already existing in Agda), and;
- (as you point out below), some things with `-AST` suffixes, to avoid conflicts with things in the "old" style.

I wonder if we should do the following in branch `main-AST`:

- comment out all code definitions and proofs in the "old style", with appropriate `TODO-2` comments indicating that they need to be migrated to the new style, perhaps with a pointer to a summary of the differences between the old and new styles
- establish good, consistent naming conventions with the small handful of existing proofs/definitions in the new style
- intend to migrate over time, and eventually merge `main-AST` into `main`

The main downside I see with this is that we don't have the resources or external contribution to actually achieve such migration at the moment. At least it would establish clear and clean intent for when/if we do have such resources.

Thoughts?  @haroldcarr @cwjnkins